File: зад3.js
function re(number){
let result = 0;
//Переворачиваем
while (number != 0) {
    result = result * 10 + (number % 10);
    //Деление без остатка
   number= Math.floor(number / 10);
}
return result;
}
//Главная
let n = prompt("Введите количество: ");
//Ввод чисел
for (let i = 0; i < n; i++) {
    let number = prompt("Введите число: ");
    //Преравниваем
    number=re(number);
    console.log(number);
}

File: зад3.rb
# Функция для переворота числа
def Re(number)
    result = 0
    while number != 0
        result = result * 10 + (number % 10)
        #Деление без остатка
        number= number / 10
    end
    return result
end

print "Введите кол-во: "
n = gets.chomp.to_i
for i in 0..n-1
print "\nВведите число: "
number = gets.chomp.to_i
#Преравниваем
number=Re(number)
print(number)
end
